### ISSUE TITLE
SuperCollider MacOS Big Sur Beta Support

### ISSUE THREAD

**Issue by IamMrandrew** [supercollider/supercollider#5160]
## Environment

* SuperCollider version: 3.11.1
* Operating system: macOS 11.0 beta 6
* Other details (Qt version, audio driver, etc.):

## Steps to reproduce

```supercollider
Paste code here. Use a minimal example please.
```

## Expected vs. actual behavior

# Exepcted
Open successfully without bugs, or at least can open it without hang

# Actual Behavior
It hanged crashed when I tried to open SuperCollider.app 
Here is the [crash report](https://gist.github.com/IamMrandrew/2f9b85cef0599495ead2c4599b5399a5.js)

Is there any updating working on to support big sur? Or is it only my personal problem on macOS big sur  getting hanged with SuperCollider?

Thanks a lot

**Comment by brianlheim** [comment]
we don't yet support this OS. we will once it's out of beta. i believe @joshpar has tried compiling SC on it.

you should not expect an official release of SuperCollider to work on Big Sur.

**Comment by lasconcore** [comment]
Any update on this?

**Comment by brianlheim** [comment]
no

**Comment by dbuendia** [comment]
> we don't yet support this OS. **we will once it's out of beta**. i believe @joshpar has tried compiling SC on it.
> 
> you should **not expect an official release** of SuperCollider to work on Big Sur.

Hi @brianlheim  

May I ask what did you mean by that? An unofficial release? 🤔 Thanks!

**Comment by fabiomorreale** [comment]
Now, this is weird. Yesterday I installed Big Sur on my main machine and SuperCollider 3.11.1 did not work - as expected. Then I try running SuperCollider 3.10.2 on older machine in which I had installed a Big Sur beta.. and it works! I copied that SC 3.10.2 on my main machine, the one with the official Big Sur release: it works perfectly. Can't really explain this, not sure if it has to do with the SuperCollider version or what.
